fenix/app/src/main/res/values/strings.xml

```diff
@@ -14,8 +14,10 @@
     <string name="content_description_private_browsing_button">Enable private browsing</string>
     <!-- Content description (not visible, for screen readers etc.): "Private Browsing" menu button. -->
     <string name="content_description_disable_private_browsing_button">Disable private browsing</string>
-    <!-- Placeholder text shown in the search bar before a user enters text -->
+    <!-- Placeholder text shown in the search bar before a user enters text for the default engine -->
     <string name="search_hint">Search or enter address</string>
+    <!-- Placeholder text shown in the search bar before a user enters text for a general engine -->
+    <string name="search_hint_general_engine">Search the web</string>
```

Comment:
we have [matching string](https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/feature/search/src/main/res/values/strings.xml#13) in A-C, but the usage is very different, didn't want to create a dependency. is there way I could speed up the translating process? We already have all the translations.

Reply:
Maybe @Delphine can help move some translations around.

Reply:
Thanks for flagging. We typically don't copy translations over, especially in mobile (see our dev best practices [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#dont-reuse-strings-in-different-contexts)).
Please land the string in Fenix and adapt the comment to its usage.